frontend/.env.example

```
@@ -1,138 +1,73 @@
-ENABLE_ADMIN_API=
-INTERNAL_JANITOR_MIN_INTERVAL_SECONDS=
-INTERNAL_JANITOR_SECRET=
-JANITOR_URL=
 
-# --- Quiz
-QUIZ_ENCRYPTION_KEY=
+STRIPE_MODE=
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=
+
+NEXT_PUBLIC_SITE_URL=
+NEXT_PUBLIC_SITE_URL=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NEXT_PUBLIC_SITE_URL key is duplicated

(DuplicatedKey)